tests/virt_autotest/kubevirt_tests_agent.pm

```diff
@@ -12,13 +12,16 @@ use strict;
 use warnings;
 use testapi;
 use lockapi;
+use transactional;
 use utils;
+use version_utils qw(is_sle_micro);
+use Utils::Systemd;
 use Utils::Backends 'use_ssh_serial_console';
 
 sub run {
     my ($self) = shift;
 
-    if (get_required_var('WITH_SLE_INSTALL')) {
+    if (get_var('WITH_SLE_INSTALL') || get_var('WITH_SLEM_INSTALL')) {
```

Comment:
Are WITH_SLE_INSTALL and WITH_SLEM_INSTALL indicators for host os? If yes, maybe is_sle/is_sle_micro are more common way?

Reply:
This parameter is just used for controlling if the test run with fresh OS installation or running on an installed system.

Reply:
Then I think you should use WITH_HOST_INSTALL.

Reply:
Updated. Thank you for suggestion.